yt_idv/shaders/ray_tracing.frag.glsl

```diff
@@ -46,8 +80,18 @@ void main()
     // This will help solve the left/right edge issues.
 
     vec3 idir = 1.0/dir;
-    vec3 tl = (left_edge - camera_pos)*idir;
-    vec3 tr = (right_edge - camera_pos)*idir;
+    vec3 tl, tr, dx_cart;
+    #ifdef NONCARTESIAN_GEOM
+    tl = (left_edge_cart - camera_pos)*idir;
```

Comment:
I'm not 100% sure but I *think* that this may be the source of the (mild!) artifacts we see when we viz `("index", "ones")` with projective integration.  I attached just-off-axis and almost-on-axis to demonstrate.

To be clear, these artifacts are not deal breakers!  But I'm wondering if this is a source of the artifacts, and if we could use more precise values to compute them.

![spherical-vr-off-edge](https://github.com/user-attachments/assets/be0af722-7604-47bc-8b9c-bbf12db94d12)
![spherical-vr-on-axis](https://github.com/user-attachments/assets/8410adf3-e5ee-46d4-90b3-07f5a9bc3140)

Reply:
it could be! and i've banged my head on many things looking for the source of those artifacts but it didn't occur to me that it might a precision issue.

Reply:
> I'm not 100% sure but I think that this may be the source of the (mild!) artifacts we see when we viz

You were 99% right! it was `dx_cart` (a few lines after this), see https://github.com/yt-project/yt_idv/pull/159#issuecomment-2839260924